chrome/content/zotero/xpcom/uri.js

```diff
@@ -172,7 +188,7 @@ Zotero.URI = new function () {
 			return path;
 		}
 		
-		if (obj instanceof Zotero.Item) {
+		if (obj instanceof Zotero.Item || obj instanceof Zotero.Feed) {
```

Comment:
Is that supposed to be `Zotero.FeedItem`? In which case, it is an instance of `Zotero.Item` anyway, so this is not necessary.